src/components/side_panel/chart/building_blocks/generic_side_panel/config_panel.ts

```diff
@@ -119,6 +119,13 @@ export class GenericChartConfigPanel extends Component<Props, SpreadsheetChildEn
     });
   }
 
+  onDataSeriesReordered(indexes: number[]) {
```

Comment:
Is there a reason we have the SelectionInput return an array of index for this callback rather that the final array of ranges ? 

From ouside the component, it'd be easire to reaceive an array of range. We could then use `onDataSeriesRangesChanged` for both props `onSelectionReordered` and `onSelectionChanged`. Or is there a reason it doesn't work ?

Reply:
I did it that way because we have to reorder the ranges and the corresponding properties, and we have to differentiate when the user manually change the ranges and when he reordered it